### DISCUSSION TITLE
knob control suggestion... also  issue regarding monoseq and cable visibility on my monitors.

### DISCUSSION THREAD

**Discussion by loatus** [maximecb/noisecraft#84] [Ideas]
I was thinking that more fine control over the knob's would be a nice feature.. perhaps with extra decimal places showing.. using control + mouse drag... similar to vcv rack.

Also I am having poor contrast on the monoseq ( selected grid squares against non selected)  and the blue patch cables are not that visible with my monitor settings . I have two monitors ( quite old) which have slightly different display settings .. same browser window shows differently on each monitor.
![283cd13d-551a-40b5-bb8a-0ade3985b7a1](https://user-images.githubusercontent.com/30633486/168288415-1a6efef9-617e-41e0-bc30-f47f0b66e2d4.jpg)
![ac8cc5c2-6ce6-40fc-8aab-aa384071ee28](https://user-images.githubusercontent.com/30633486/168288429-cc886bb4-ffae-48da-b673-cdbfaad7f157.jpg)

,thanks

**Comment by maximecb** [comment]
For finer control on the knobs, you should be able to hold shift while doing click and drag.

I'll see what I can do about increasing the contrast on the grid cells :)

**Comment by loatus** [comment]
didn't realize about shift n drag... thanks very much with the contrast
issue, greatly appreciated. 😎

**Comment by maximecb** [comment]
I increased the contrast between the on and off grid cells a bit.

**Reply by loatus** [reply]
thats much better, thanks
